src/__tests__/downshift.props.js

```diff
@@ -129,7 +129,6 @@ test('uses given environment', () => {
   wrapper.unmount()
   expect(environment.addEventListener).toHaveBeenCalledTimes(2)
   expect(environment.removeEventListener).toHaveBeenCalledTimes(2)
-  expect(environment.document.getElementById).toHaveBeenCalledTimes(1)
```

Comment:
This function is no longer called, as its called in the `componentDidUpdate` lifecycle.